### ISSUE TITLE
HandBrake Preset Object

### ISSUE THREAD

**Issue by EpicKris** [75lb/handbrake-js#63]
To be able to get a preset object.

HandBrakeCLI Command: `HandBrakeCLI --preset-export "Apple 2160p60 4K HEVC Surround"`
```json
{
    "PresetList": [
        {
            "AlignAVStart": false,
            "AudioCopyMask": [
                "copy:aac",
                "copy:ac3",
                "copy:eac3",
                "copy:dtshd",
                "copy:dts",
                "copy:mp3",
                "copy:truehd",
                "copy:flac"
            ],
            "AudioEncoderFallback": "ca_aac",
            "AudioLanguageList": [],
            "AudioList": [
                {
                    "AudioBitrate": 128,
                    "AudioCompressionLevel": -1.0,
                    "AudioDitherMethod": "auto",
                    "AudioEncoder": "ca_aac",
                    "AudioMixdown": "dpl2",
                    "AudioNormalizeMixLevel": false,
                    "AudioSamplerate": "auto",
                    "AudioTrackDRCSlider": 0.0,
                    "AudioTrackGainSlider": 0.0,
                    "AudioTrackQuality": -1.0,
                    "AudioTrackQualityEnable": false
                }
            ],
            "AudioSecondaryEncoderMode": true,
            "AudioTrackSelectionBehavior": "first",
            "ChapterMarkers": true,
            "ChildrenArray": [],
            "Default": true,
            "FileFormat": "av_mp4",
            "Folder": false,
            "FolderOpen": false,
            "InlineParameterSets": false,
            "Mp4HttpOptimize": false,
            "Mp4iPodCompatible": false,
            "PictureAutoCrop": true,
            "PictureBottomCrop": 0,
            "PictureChromaSmoothCustom": "",
            "PictureChromaSmoothPreset": "off",
            "PictureChromaSmoothTune": "none",
            "PictureCombDetectCustom": "",
            "PictureCombDetectPreset": "off",
            "PictureDARWidth": 0,
            "PictureDeblockCustom": "strength=strong:thresh=20:blocksize=8",
            "PictureDeblockPreset": "off",
            "PictureDeblockTune": "medium",
            "PictureDeinterlaceCustom": "",
            "PictureDeinterlaceFilter": "off",
            "PictureDeinterlacePreset": "default",
            "PictureDenoiseCustom": "",
            "PictureDenoiseFilter": "off",
            "PictureDenoisePreset": "medium",
            "PictureDenoiseTune": "none",
            "PictureDetelecine": "off",
            "PictureDetelecineCustom": "",
            "PictureForceHeight": 0,
            "PictureForceWidth": 0,
            "PictureHeight": 0,
            "PictureItuPAR": false,
            "PictureKeepRatio": true,
            "PictureLeftCrop": 0,
            "PictureLooseCrop": false,
            "PictureModulus": 2,
            "PicturePAR": "loose",
            "PicturePARHeight": 720,
            "PicturePARWidth": 853,
            "PictureRightCrop": 0,
            "PictureRotate": "angle=0:hflip=0",
            "PictureSharpenCustom": "",
            "PictureSharpenFilter": "off",
            "PictureSharpenPreset": "medium",
            "PictureSharpenTune": "none",
            "PictureTopCrop": 0,
            "PictureWidth": 0,
            "PresetDescription": "",
            "PresetName": "Apple 2160p60 4K HEVC Surround",
            "SubtitleAddCC": false,
            "SubtitleAddForeignAudioSearch": false,
            "SubtitleAddForeignAudioSubtitle": false,
            "SubtitleBurnBDSub": false,
            "SubtitleBurnBehavior": "none",
            "SubtitleBurnDVDSub": false,
            "SubtitleLanguageList": [],
            "SubtitleTrackSelectionBehavior": "none",
            "Type": 0,
            "UsesPictureFilters": true,
            "UsesPictureSettings": 0,
            "VideoAvgBitrate": 6000,
            "VideoColorMatrixCodeOverride": 0,
            "VideoEncoder": "x264",
            "VideoFramerate": "auto",
            "VideoFramerateMode": "vfr",
            "VideoGrayScale": false,
            "VideoLevel": "auto",
            "VideoOptionExtra": "",
            "VideoPreset": "medium",
            "VideoProfile": "auto",
            "VideoQSVAsyncDepth": 4,
            "VideoQSVDecode": false,
            "VideoQualitySlider": 22.0,
            "VideoQualityType": 2,
            "VideoScaler": "swscale",
            "VideoTune": "",
            "VideoTurboTwoPass": false,
            "VideoTwoPass": false,
            "x264Option": "",
            "x264UseAdvancedOptions": false
        }
    ],
    "VersionMajor": 42,
    "VersionMicro": 0,
    "VersionMinor": 0
}
```

**Comment by rupamking1** [comment]
@EpicKris 
can u please tell me how to setup pre-set in nodejs

**Comment by 75lb** [comment]
We need to add `preset-export` to the [cli options](https://github.com/75lb/handbrake-js/blob/master/lib/cli-options.js#L17). While there, check if any other new HandbrakeCLI options are missing.

**Comment by 75lb** [comment]
support for `--preset-export` released in v5.0.3.
